cloudoperators/juno
# chore(carbon): enable PR preview for carbon app

# Summary

The purpose of these changes are to enable PR preview for carbon app.

# Changes Made



- Enabled PR preview for carbon app
- Display **Home page!**  when clicked on greenhouse logo. This is just to test if the PR preview is working correctly.

# Testing Instructions



- Make sure you're able to see the preview on this PR and when clicked on greenhouse logo in the carbon app you're able to see the **Home page!** text.

# Checklist
